samples/sample_capture_async.py

```diff
@@ -0,0 +1,61 @@
+"""Sample demonstrating how to capture with multiple cameras at the same time."""
+import concurrent.futures
+import datetime
+import time
+
+import zivid
+
+
+def _sync_version(cameras: list[zivid.Camera]) -> list[zivid.Frame]:
+    return [
+        camera.capture(
+            zivid.Settings(
+                acquisitions=[
+                    zivid.Settings.Acquisition(
+                        exposure_time=datetime.timedelta(microseconds=100000)
+                    )
+                ]
+            )
+        )
+        for camera in cameras
+    ]
+
+
+def _async_version(cameras: list[zivid.Camera]) -> list[zivid.Frame]:
+    with concurrent.futures.ThreadPoolExecutor() as executor:
+        futures = [
+            executor.submit(
+                camera.capture,
+                zivid.Settings(
+                    acquisitions=[
+                        zivid.Settings.Acquisition(
+                            exposure_time=datetime.timedelta(microseconds=100000)
+                        )
+                    ]
+                ),
+            )
+            for camera in cameras
+        ]
+
+        return [future.result() for future in futures]
+
+
+def _main():
+    app = zivid.Application()
+    cameras = app.cameras()
+    for camera in cameras:
+        camera.connect()
+
+    start = time.monotonic()
+    _async_version(cameras)
+    end = time.monotonic()
+    print("Async time:", end - start, "seconds")
```

Comment:
This text is a bit lacking of context, maybe it can say "Time taken to capture asynchronously from X cameras: Y seconds".

Reply:
Amended